ephios/extra/permissions.py

```diff
@@ -39,7 +39,7 @@ class PermissionField(BooleanField):
 
     def __init__(self, *args, **kwargs):
         self.permission_set = kwargs.pop("permissions")
-        super().__init__(*args, **kwargs)
+        super().__init__(*args, required=kwargs.pop("required", False), **kwargs)
```

Comment:
This makes it default to not being required... Which makes sense for this use case, but deviates from the default Boolean Field (which has other semantics...)

Reply:
yes, that was my intention as I think it should be an explicit choice to mark this as required and plugins could easily forget the required=False especially if they don't have the background of the PermissionField inheriting from BooleanField